scim2-sdk-common/src/main/java/com/unboundid/scim2/common/messages/ListResponse.java

```diff
@@ -51,21 +51,20 @@
   @JsonProperty(value = "totalResults", required = true)
   private final int totalResults;
 
-
```

Comment:
It might be worth using the [JsonPropertyOrder annotation](https://fasterxml.github.io/jackson-annotations/javadoc/2.8/com/fasterxml/jackson/annotation/JsonPropertyOrder.html) instead to order the properties, but since this change works with your unit test I'm fine with it.

Reply:
I went ahead and added that as well.